packages/venia-concept/src/components/Footer/footer.js

```diff
@@ -81,14 +80,8 @@ class Footer extends Component {
                                     </span>
                                 );
                             }
-                            const storeName =
-                                data.storeConfig.name || 'Magento';
 
-                            return (
-                                <span>
-                                    © {storeName} {year}. All rights reserved.
-                                </span>
-                            );
+                            return <span>{data.storeConfig.copyright}</span>;
```

Comment:
If this is undefined that's ok?

Reply:
GraphQL will always return a string or null, undefined should not be possible.

Reply:
Yea, I think I meant that maybe we should not even render a span.